src/plugins/intel_gpu/src/plugin/ops/roll.cpp

```diff
@@ -0,0 +1,75 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "ngraph/op/roll.hpp"
+
+#include "intel_gpu/plugin/common_utils.hpp"
+#include "intel_gpu/plugin/program.hpp"
+#include "intel_gpu/primitives/roll.hpp"
+
+namespace ov {
+namespace runtime {
+namespace intel_gpu {
+
+namespace {
+
+void CreateRollOp(Program& p, const std::shared_ptr<ngraph::op::v7::Roll>& op) {
+    p.ValidateInputs(op, {3});
+
+    auto inputs = p.GetInputPrimitiveIDs(op);
+    auto layer_name = layer_type_name_ID(op);
+    auto input_shape = op->get_input_shape(0);
+    uint8_t rank = input_shape.size();
+    auto format = DefaultFormatForDims(rank);
+    auto default_rank = format.dimension();
+
+    auto shift_constant = std::dynamic_pointer_cast<ngraph::op::Constant>(op->get_input_node_shared_ptr(1));
+    if (!shift_constant) {
+        IE_THROW() << "Unsupported parameter node type in " << op->get_friendly_name() << " (" << op->get_type_name()
```

Comment:
op->get_friendly_name()  is used several times in this function. Maybe use dedicated variable for it?

Reply:
done